<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfiCamera</class>
 <widget class="QWidget" name="ConfiCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>235</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>235</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configurar Camara</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ConfiCamera{
	background-color: #eff9fe;
}
#frmContainer{
	background-color: white;
	border-radius: 10px;
}

QPushButton{
	background-color: #3838FF;
	padding: 5px;
	border-radius: 5px;
	color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frmContainer">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lblIP">
           <property name="text">
            <string>IP:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leIp"/>
         </item>
         <item>
          <widget class="QLabel" name="lblUser">
           <property name="text">
            <string>Usuario:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leUser"/>
         </item>
         <item>
          <widget class="QLabel" name="lblPassword">
           <property name="text">
            <string>Contrase√±a:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnSave">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Guardar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
